src/containers/LivingWhitepaper/LivingWhitepaper.scss

```diff
@@ -0,0 +1,155 @@
+@import 'styles/font-mixins';
+
+.LivingWhitepaper {
+  &__hero {
+    align-items: center;
+    background: url('./assets/LivingWhitepaperBg.webp');
+    background-position: center;
+    background-size: cover;
+    display: flex;
+    height: 248px;
+    justify-content: space-between;
+    padding: 48px 74px;
+
+    @media (max-width: 768px) {
+      flex-direction: column;
+      height: fit-content;
+      padding: 45px 40px;
```

Comment:
@jamessspanggg I'd suggest you to follow 8 pt or 10 pt grid for the designs. So, these values will be multiple of 8 mostly. 10, if 8 doesn't look good. 4, if the design is too dense and has a lot of elements.

Reply:
let me look into the concept of 8pt or 10pt grid....